go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/johnfercher/maroto v0.37.0
 	github.com/kubescape/go-logger v0.0.6
 	github.com/kubescape/k8s-interface v0.0.83
-	github.com/kubescape/opa-utils v0.0.188
+	github.com/kubescape/opa-utils v0.0.190
```

Comment:
We have to bump the version of `opa-utils` to the latest before the tests can pass and we implement the needed functionality.

So, this PR depends on kubescape/opa-utils#33 being published.